setup.cfg

```diff
@@ -31,16 +31,16 @@ include_package_data = True
 packages = find:
 scripts =
   bin/generate_schema
-  bin/patch_content_types
-  bin/use_external_components
 install_requires =
     django>=3.2.0
     django-filter>=2.0
     django-solo
     djangorestframework>=3.11.0
+    djangorestframework-gis>=1.0
```

Comment:
How were fields that use this library handled previously? I don't know if we want all the projects that are using this library to install this if it won't be used for for anything :thinking: .